pkg/frontend/mysql_cmd_executor.go

```diff
@@ -133,6 +136,29 @@ func (mce *MysqlCmdExecutor) GetRoutineManager() *RoutineManager {
 	return mce.routineMgr
 }
 
+func (mce *MysqlCmdExecutor) RecordStatement(ctx context.Context, ses *Session, proc *process.Process, sql string, beginIns time.Time) context.Context {
+	statementId := util.Fastrand64()
+	sessInfo := proc.SessionInfo
+	trace.ReportStatement(
+		ctx,
+		&trace.StatementInfo{
+			StatementID:          statementId,
+			SessionID:            sessInfo.GetConnectionID(),
+			TransactionID:        ses.GetTxnHandler().GetTxn().GetID(),
+			Account:              "account",
```

Comment:
make this as sessionInfo.GetAccount(), even if we return a dummy value there.

Reply:
got it.